tools/dart-api-builder/services/packageInfo.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('canonical-path');
+
+/**
+ * Load information about this project from the pubspec.yaml
+ * @return {Object} The package information
+ */
+module.exports = function packageInfo() {
+  const ngPath = '../angular-dart';
```

Comment:
I agree, that bothered me too, but it's hard coded in the same way on the TS side :)
See `tools/api-builder/angular.io-package/services/packageInfo.js` and that's good enough for me (for now).